src/frontend/src/components/edit-profile.js

```diff
@@ -402,7 +402,7 @@ class EditProfile extends PureComponent<Props> {
             html
             effect="solid"
             place="top"
-            offset='{"top": -10}'
+            offset={{ top: -10 }}
```

Comment:
Thanks for fixing this! It looks like this fixes issue #519 so we should add a comment to that issue and drag it into review.

Reply:
Good call. Thanks!